MonstieWash/Assets/Scripts/Player/PlayerHand_AnimationController.cs

```diff
@@ -0,0 +1,86 @@
+using NUnit.Framework.Constraints;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerHand_AnimationController : MonoBehaviour
```

Comment:
Class names should not include underscores as per C# convention. I know it's a long name but it is what it is.
```suggestion
public class PlayerHandAnimationController : MonoBehaviour
```

Reply:
Will change when I'm back home, thanks